src/main/resources/config/en.yml

```diff
@@ -94,29 +99,37 @@ Experiment_MCDiscordReserializer_ToMinecraft: false
 Experiment_MCDiscordReserializer_InBroadcast: false
 
 # Other
+# Whether to cancel a console command if it can't be logged
 CancelConsoleCommandIfLoggingFailed: true
+# A language code to force DiscordSRV to use; See https://github.com/DiscordSRV/DiscordSRV/tree/master/src/main/resources/config for a list of available codes
 ForcedLanguage: none
+# Whether to print a list of guilds and their channels to the console on start
 PrintGuildsAndChannels: true
+# Whether to force the use of TLSv12
 ForceTLSv12: true
+# Returns valid hostname without making an attempt to actually validate it (insecure)
 NoopHostnameVerifier: false
+# How many times DiscordSRV will try to use your system's DNS before using CloudFlare and Google DNS servers instead
 MaximumAttemptsForSystemDNSBeforeUsingFallbackDNS: 3
```

Comment:
This actually defines how many times it will let system dns fail before completely switching to only using fallback dns. 

Additionally 0 can be used to always fallback, and negative values are suppose to never fallback, [but the relevant chek](https://github.com/DiscordSRV/DiscordSRV/blob/339e5e81da67cfe6210679af775eb66a9b5033be/src/main/java/github/scarsz/discordsrv/DiscordSRV.java#L743) is wrong (should be `< 0`)

Additionally unrelated to the documentation, we probably shouldn't be setting this to a positive value since the user might not accept of the use of or the terms of Google's/Cloudflare's for these services.